Library/Formula/libphonenumber.rb

```diff
@@ -0,0 +1,37 @@
+class Libphonenumber < Formula
+  desc "C++ Phone Number library by Google"
+  homepage "https://github.com/googlei18n/libphonenumber"
+  url "https://github.com/googlei18n/libphonenumber/archive/libphonenumber-7.2.3.tar.gz"
+  sha256 "9f93615b23d8a12a42099d8113a8d6304278bee4d0ef09fd366188a206aacd46"
+
+  depends_on "cmake" => :build
+  depends_on :java => "1.7+"
+  depends_on "icu4c"
+  depends_on "protobuf"
+  depends_on "boost"
+  depends_on "re2"
+
+  patch :DATA
```

Comment:
Could you please add a comment above this line explaining why this patch is necessary? Thanks!